doxia-core/src/test/java/org/apache/maven/doxia/sink/impl/SinkEventElement.java

```diff
@@ -80,4 +80,22 @@ public String toString() {
         builder.append(']');
         return builder.toString();
     }
+
+    @Override
+    public int hashCode() {
```

Comment:
These methods could use tests

Reply:
Those are test classes only! I don't think adding test for tester utilities is very useful. Tests will fail if those don't work!